source/plugins/toolbox-plugin.rst

```diff
@@ -1,22 +1,19 @@
 ToolBox interface
 =================
 
-ToolBox is a simple HTTP user interface allowing to use some features when no GLPI server is available.
+**ToolBox** is a simple web interface embedded into GLPI Agent that allows users to configure some features when there is no GLPI server available. For example:
+
+- When you have an isolated network that cannot or should not be connected to a GLPI Server or a GLPI Proxy :doc:`../proxy-server-plugin` for any reason but you still need to run a network discovery or a network inventory.
```

Comment:
There's a mistake here on the documentation ref. It is indeed at the same level and the syntax set the text from the page title:
```suggestion
- When you have an isolated network that cannot or should not be connected to a GLPI Server or via GLPI Agent :doc:`proxy-server-plugin` for any reason but you still need to run a network discovery or a network inventory.
```